glhe/input_processor/input_processor.py

```diff
@@ -1,4 +1,6 @@
+from json import loads
 import os
```

Comment:
Looks like another spot to get rid of `os`.  It's just being used to join paths in one function below.